### ISSUE TITLE
About Classification Results

### ISSUE THREAD

**Issue by wl1282215100** [gallegi/T4E_MICCAI_BrainTumor#3]
I came across your RSNA-ASNR-MICCAI project code on GitHub and found it very interesting. While trying to review your code, I encountered a few issues and I am reaching out to seek your help in addressing them.

I attempted to reproduce your code for segmentation and classification, but since there were no labels available for the test set, I split the training set into a new train and test set to evaluate the performance of the trained models. However, when I tried to reproduce the classification part of the code, the results I obtained had an AUC of only around 0.5 and accuracy was also around 0.5. I was unable to achieve the same level of accuracy as the results you presented on GitHub. Despite several attempts, I have been unable to identify the source of the issue.

I was wondering if you could let me know if the code you used to obtain the final results has been updated on GitHub, If so, would it be possible for you to share the updated version of your code? And if you think the dataset itself may not have clear classification indicators, leading to poor model performance during training.

Appreciate your help.
Thanks and regards.
